### PR TITLE
Laue dials preprint blogpost

### PR DESCRIPTION
**Pull request by PrinceWalnut**
Blogpost about the new Laue-DIALS preprint on bioRxiv, including a few specifics of the package and figure illustrations about use + pink-beam crystallography.

### CODE REVIEW

**Comment by PrinceWalnut** [comment]
@dennisbrookner could you test this works in Jekyll just fine and will display appropriately? I used Kevin's suggestions for displaying media but want to be sure

File: _posts/2024-08-07-laue-dials-preprint.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+{:image: style="margin-right: 25%;margin-left: 25%;width: 50%"}
+{:caption: style="display: inline-block;padding: 0 15%;"}
+
+![Reciprocal lattice points in the Laue case](/assets/posts/2024-08-07-laue-dials-preprint.md/laue_rlp.png){: image}
+*Full reflections from Laue pink-beam crystallography vs monochromatic stills and rotations from [Hewitt et al](https://www.biorxiv.org/content/10.1101/2024.07.23.604358v1). [(CC-BY-NC license)](https://creativecommons.org/licenses/by/4.0/)*{: caption}
+
+Laue-DIALS is currently able to handle single-crystal fixed-target datasets, with development to extend to serial crystallography rapidly underway! Installing Laue-DIALS just requires a [DIALS installation](https://dials.github.io/installation.html) and then running `pip install laue-dials`. All of the analyses in the preprint were performed using `DIALS v3.17.0` and `Laue-DIALS v0.4`.
```
Comment (kmdalton) [inline _posts/2024-08-07-laue-dials-preprint.md:18]: Suggest the following:

Laue-DIALS is currently able to handle single-crystal fixed-target datasets, with development to extend to serial crystallography rapidly underway! Laue-DIALS can be installed using `pip` after [installing the DIALS diffraction analysis package](https://dials.github.io/installation.html).

File: _posts/2024-08-07-laue-dials-preprint.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+*Full reflections from Laue pink-beam crystallography vs monochromatic stills and rotations from [Hewitt et al](https://www.biorxiv.org/content/10.1101/2024.07.23.604358v1). [(CC-BY-NC license)](https://creativecommons.org/licenses/by/4.0/)*{: caption}
+
+Laue-DIALS is currently able to handle single-crystal fixed-target datasets, with development to extend to serial crystallography rapidly underway! Installing Laue-DIALS just requires a [DIALS installation](https://dials.github.io/installation.html) and then running `pip install laue-dials`. All of the analyses in the preprint were performed using `DIALS v3.17.0` and `Laue-DIALS v0.4`. 
+
+### Quick Tutorial
+Laue-DIALS has a variety of tutorial notebooks. Each analysis in the preprint was performed with a notebook that can be found [in the associated deposition on Zenodo](https://zenodo.org/records/12761162). A data download script is provided to download any desired raw data from [SBGrid](https://sbgrid.org) (simulated data is high-volume and available on request!). A routine data analysis pipeline is described by the following figure:
```
Comment (kmdalton) [inline _posts/2024-08-07-laue-dials-preprint.md:21]: suggest:

Laue-DIALS has a variety of tutorial notebooks. Each analysis in the preprint was performed with a notebook that can be found [in the associated deposition on Zenodo](https://zenodo.org/records/12761162). Experimental data are available from [SBGrid](https://sbgrid.org). A routine data analysis pipeline is described by the following figure:

**Review by kmdalton — APPROVED** [review]
looks good. i made some stylistic suggestions.

**Review by dennisbrookner — APPROVED** [review]
This looks good to me. I just made the following minor changes:

 - reformat the blog post to use the `.blog-XXX` CSS classes
 - create the new CSS class `.blog-image-narrow` which has 50% width, rather than 80% width. I think this will continue to be useful going forward
 - remove the unnecessary `usemathjax` from the front matter

**Comment by PrinceWalnut** [comment]
Okay, implemented Kevin's suggestions for the phrasing. @dennisbrookner I'll leave it to you for a final approval/merge if everything formats correctly.

**Pull request merged**